mscore/preferences.cpp

```diff
@@ -1424,6 +1442,13 @@ void PreferenceDialog::apply()
 
 #ifdef USE_PORTMIDI
       prefs.portMidiInput = portMidiInput->currentText();
+      prefs.portMidiOutput = portMidiOutput->currentText();
+      if (static_cast<PortMidiDriver*>(static_cast<Portaudio*>(seq->driver())->mididriver())->isSameCoreMidiIacBus(prefs.portMidiInput, prefs.portMidiOutput)) {
+            QMessageBox msgBox;
+            msgBox.setText("Warning: You used the same CoreMIDI IAC bus for input and output.  This will cause problematic loopback of MuseScore's output midi messages, since they will be sent back in to MuseScore as input, causing confusion.  To avoid this problem, use Mac OS's Audio MIDI Setup to create a dedicated virtual port for MuseScore's Midi output, restart MuseScore, and then select that port as output.  Then other programs may use that deidcated virtual port to receive MuseScore's midi input.");
```

Comment:
ok, well I'm going with via Spotlight, since I would presume that the person would search by typing.  Anyway, I'm not going to analyze this too much more.  The latest push from a minute ago has:

"Warning: You used the same CoreMIDI IAC bus for input and output.  This will cause problematic loopback, whereby MuseScore's outputted MIDI messages will be sent back to MuseScore as input, causing confusion.  To avoid this problem, access Audio MIDI Setup via Spotlight to create a dedicated virtual port for MuseScore's MIDI output, restart MuseScore, return to Preferences, and select your new virtual port for MuseScore's MIDI output.  Other programs may then use that dedicated virtual port to receive MuseScore's MIDI output."